docs/install-azure-repos.md

```diff
@@ -0,0 +1,230 @@
+[Go back to the main documentation page](../README.md)
+
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
+
+**Step 1:** Make sure you have the connection details of your JFrog environment.
```

Comment:
Let's add a new-line between the step title and the step test and also replace the "bold" wrapping around the step number with a "title" markdown annotation - ```## Step 1``` . The above therefore should be changed to - 
## Step 1
Make sure you have the connection details of your JFrog environment.

Reply:
done